recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "optuna" %}
-{% set version = "3.0.4" %}
+{% set version = "3.0.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: f3ce3126c40a2d8ec0a290861a622a91d0a73698942157a2589bbe648225134d
+  sha256: 9e39869e76c8b57b7b0f9badb31853fffed29e1da52b508fd78bd1d5307b79c8
```

Comment:
I confirmed the hash value is correct.
See https://pypi.org/project/optuna/3.0.5/#copy-hash-modal-4411e225-b52f-4544-af1f-95ef2b3a803f.